project_scrum/models/project_scrum_us.py

```diff
@@ -0,0 +1,164 @@
+            ('questionable', 'Questionable'),
+            ('reverse', 'Reverse'),
+            ('not_set', 'Not Set'),
+        ]
+
+    name = fields.Char(string='User Story', required=True)
+    color = fields.Integer(related='project_id.color')
+    description = fields.Html(string='Description',)
+    actor_ids = fields.Many2many(
+        comodel_name='project.scrum.actors',
+        string='Actor',
+    )
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+        ondelete='set null',
+        index=True,
+        track_visibility='onchange',
+        change_default=True,
+    )
+    sprint_ids = fields.Many2many(
+        comodel_name='project.scrum.sprint',
+        string='Sprint',
+    )
+    task_ids = fields.One2many(
+        comodel_name='project.task',
+        inverse_name='us_id',
+    )
+    task_test_ids = fields.One2many(
+        comodel_name='project.scrum.test',
+        inverse_name='user_story_id_test',
+    )
+    task_count = fields.Integer(compute='_compute_task_count', store=True)
+    test_ids = fields.One2many(
+        comodel_name='project.scrum.test',
+        inverse_name='user_story_id_test',
+    )
+    test_count = fields.Integer(compute='_compute_test_count', store=True)
+    sequence = fields.Integer()
+    company_id = fields.Many2one(
+        related='project_id.analytic_account_id.company_id',
+    )
+    moscow = fields.Selection('_get_moscow_field', string='Moscow')
+    value = fields.Selection('_get_value_field', string='Value')
+    risk = fields.Selection('_get_risk_field', string='Risk')
+    kano = fields.Selection('_get_kano_field', string='Kano')
+    reference = fields.Char(
+        'Number',
+        index=True,
+        readonly=True,
+        copy=False,
+        default='/',
+    )
+    kanban_state = fields.Selection([
+        ('normal', 'Mark as impeded'),
+        ('blocked', 'Mark as waiting'),
+        ('done', 'Mark item as defined and ready for implementation')],
+        'Kanban State',
+        default='blocked',
+    )
+
+    @api.depends("task_ids")
+    def _compute_task_count(self):
+        for p in self:
+            p.task_count = len(p.task_ids)
+
+    @api.depends("test_ids")
+    def _compute_test_count(self):
+        for p in self:
+            p.test_count = len(p.test_ids)
+
+    def _resolve_project_id_from_context(self):
+        """ Returns ID of project based on the value of 'default_project_id'
+            context key, or None if it cannot be resolved to a single
+            project.
+        """
+        context = self.env.context
+        if type(context.get('default_project_id')) in (int, long):
+            return context['default_project_id']
+        if isinstance(context.get('default_project_id'), basestring):
+            project_name = context['default_project_id']
+            project_ids = self.env['project.project'].with_context(
+                context).name_search(name=project_name)
```

Comment:
pass `operator` too, the default `ilike` won't yield what we want, or only fall back to that when `=` or `=ilike` don't yield results